hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/SCMNodeManager.java

```diff
@@ -124,6 +124,8 @@ public class SCMNodeManager implements NodeManager {
   private final boolean useHostname;
   private final ConcurrentHashMap<String, Set<String>> dnsToUuidMap =
       new ConcurrentHashMap<>();
+  private final Map<String, Set<String>> addressToUuidMap =
```

Comment:
If this map is used to keep both ip and host names, probably we can deprecate the old dnsToUuidMap. It would be nice if you can clean up the code.